python/cuML/knn/c_knn.pxd

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+cimport numpy as np
+from libcpp cimport bool
+np.import_array
+
+cdef extern from "knn/knn.cu" namespace "ML":
+    pass 
+
+cdef extern from "knn/knn_c.h" namespace "ML":
+    cdef cppclass kNN:
+        kNN(int D) except +
+        void search(float *search_items, int search_items_size, long *res_I, float *res_D, int k)
```

Comment:
could we use the one line per parameter formatting we have in the other methods? Just makes it easier to read/debug/use later.

Reply:
Sure, I don't mind making that a standard.